src/backend/joanie/tests/core/test_api_order.py

```diff
@@ -776,13 +816,16 @@ def test_api_order_create_authenticated_invalid_product(self):
             response.json(),
             {
                 "__all__": [
-                    'The product "balançoire" is not linked to course "mathématiques".'
+                    'The course "mathématiques" and the organization "fun" '
+                    'should be linked to the product "balançoire".'
                 ]
             },
         )
```

Comment:
as it is a generic error for multiple tests, I would add the expected behaviour just after to confort the test and not to doubt the error is really because of the missing organization

```suggestion
        )
        # we add the organization to the product
        product.organizations.add(organization)
        response = self.client.post(
            "/api/v1.0/orders/",
            data=data,
            content_type="application/json",
            HTTP_AUTHORIZATION=f"Bearer {token}",
        )
        #now it works as expected
        self.assertEqual(response.status_code, 201)
        self.assertTrue(models.Order.objects.exists())
```

Reply:
Good idea!